Cargo.toml

```diff
@@ -230,3 +232,4 @@ zksync_shared_metrics = { path = "core/node/shared_metrics" }
 zksync_block_reverter = { path = "core/node/block_reverter" }
 zksync_commitment_generator = { path = "core/node/commitment_generator" }
 zksync_house_keeper = { path = "core/node/house_keeper" }
+zksync_shared = { path = "core/node/shared" }
```

Comment:
Bikeshedding: IMO, `zksync_shared` is too generic; `zksync_node_shared` would be a better name. The crate directory could still be called `shared` to avoid duplication.